package.json

```diff
@@ -136,4 +136,4 @@
     "typescript": "^4.3.2",
     "wait-on": "^6.0.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
omit no change files, same with package lock

Reply:
ok but shouldn't we have a empty line at the bottom?

Reply:
depends on eslint rule, if thats the case ok lo. probably this file was b4 the rule kick in, not sure